pkg/jx/cmd/add_app_test.go

```diff
@@ -622,6 +622,7 @@ func TestAddAppForGitOpsWithSecrets(t *testing.T) {
 		// Test boolean type
 		console.ExpectString("Enter a value for tokenValue")
 		console.SendLine("abc")
+		console.ExpectString(" ***")
```

Comment:
Is this fixing the random test failures? If so, any ideas why the tests were not failing all the time w/o this change?